### DISCUSSION TITLE
[META] Can the discord room be bridged to some/a matrix/IRC room?

### DISCUSSION THREAD

**Discussion by JCWasmx86** [Prince781/vala-language-server#238] [Ideas]
First of all, I can completely understand, if you don't want to do this, as I don't know how many would want this.

Would it be possible to setup the bridge between the vala discord and a matrix room? I would like to join Discord, too, but they want to have a phone number and I would consider that not acceptable, as even the support can override this. I'm sure there are a lot of other people uncomfortable to give some phone number.

**Comment by Prince781** [comment]
Hmmm...I thought I've set things up so you shouldn't need a phone number to join and interact in the Vala Discord. 

![image](https://user-images.githubusercontent.com/1341033/163656592-a1608b6b-0a40-41e5-9e89-4d57f195866a.png)

**Reply by JCWasmx86** [reply]
If I want to join over the link in the bottom of the README:

![image](https://user-images.githubusercontent.com/63594136/163663599-f8c5e3bb-b230-43e7-ba8c-35b5812dc9a5.png)


(It worked in the past, the day you merged the code action for implementing the required members - so this is probably a problem on my side, there I saw that there were several JCWasmx86's in the channel, all probably from me)

No matter what I do, it first wants my E-Mail address and then my phone number - although that is out of scope of this discussion as Discord's authentication is a horrible mess

**Reply by JCWasmx86** [reply]
Nevermind, I gave them my phone number :(, but at least I was able to join now

**Reply by Prince781** [reply]
I didn't need a phone number to register. Hmm...
